### PR TITLE
DM-31161: Create documentation for bias ScriptQueue scripts from DM-30483

### PR DESCRIPTION
**Pull request by plazas**

### CODE REVIEW

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+.. _Constructing Calibrations documentation: https://lsst.ncsa.illinois.edu/~czw/pipelines_lsst_io/_build/html/modules/lsst.cp.pipe/constructing-calibrations.html 
+
+.. _ComCam-Master-Bias-Procedure-Prerequisites:
+
+Prerequisites
+=============
+
+- You should be logged into the LSST Operations and Visualization Enviroment (LOVE) at the Summit :ref:`operational environment <Observing-Interface-Operational-Environments>`.
+- The script assumes  that the LSSTComcam and OCPS are both ``ENABLED``, and that latter has been ``ENABLED`` with the configuration of ``LSSTComCam``.
+- A `butler`_ is needed to run the pipetask bias command.
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:38]: I am not really sure I understand this point here. Does the Script needs access to butler in order to run or is it the OCPS?

In any case, I think this is a general environment consideration that is probably out of the scope of this procedure. 

I think you could remove it.

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+     Screenshot of LOVE interface with the "MTQueue" pannel.
+
+
+Load the Script
+---------------
+
+After clicking on the ``MTQueue`` panel, search for the script ``maintel/make_comcam_bias.py`` under ``AVAILABLE SCRIPTS`` on the left, as shown in the figure below:
+
+.. figure:: ./_static/love-available-scripts.png
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:69]: I think you cropped the image too much. I can't see the "add" button in the pop-up box. Also, can you add an arrow indicating the "add" button?

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+.. _collection: https://pipelines.lsst.io/v/daily/modules/lsst.daf.butler/organizing.html
+
+.. _ComCam-Master-Bias-Procedure-Steps:
+
+Procedure Steps
+===============
+
+Once you are logged into LOVE, click on the ``MTQueue`` panel, as shown in the figure below: 
+
+.. figure:: ./_static/love-mtqueue-panel.png
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:58]: can you highlight the MTQueue with a box around it or an arrow?

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+
+      Screenshot of LOVE interface with the "AVAILABEL SCRIPTS" list.
+      
+Load the script by clicking on the button in front of the name of the script that has a triangle.
+
+Enter configuration parameters
+------------------------------
+
+After loading the script, a window that contains two sections, ``SCHEMA`` (top) and ``CONFIG`` (bottom), will appear. The former will show the configuration options (and the default values of some of them) available that should be entered in the latter. The configuration options are as follows:
+
```
Comment (tribeiro) [inline Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:80]: Can you add a text code block with a configuration people can copy and paste directly?

```
.. code-block:: text

    n_bias: 2
    detectors: (0,1,2,3,4,5,6,7,8)
    ...
```

Or at least the minumum configuration people will need (I think `n_bias` is the only required parameter right?).

**Review by tribeiro — APPROVED** [review]
Looks good. 

I left a few comments inline, but nothing major. Please do take a look at them before merging.

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst (hunk 1 above)
Comment (plazas) [reply Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:38]: I was referring to the fact that the pipetask needs a repo such as ``/repo/LSSTComCam`` and that the stack should be loaded. But I think you are correct;  I'll remove it per your consideration.

File: Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst (hunk 4 above)
Comment (plazas) [reply Operational-Procedures/Main-Telescope/ComCam-Master-Bias-Procedure/comcam-master-bias-procedure.rst:80]: For this script, the required parameters are ``input_collections_bias, calib_collection, repo``. In the next iteration (I'm working on a script for biases, darks, and flats; DM-31306), I'll try to make in such a way that as many parameters as possible have defaults (following a suggestion by you). I'll add examples here and in the LATISS document,  as you suggest here.

**Comment by Slimleashma** [comment]
I would recommend that a few additional steps of clarification be added:
- Under Prerequisites, maybe clarify how to put LATISS and OCPS into enabled state. I use a notebook in conjunction to do this, but it's not obvious to a new person. This requires Chronograf or LOVE to verify the status. 
- Under entering configuration parameters, we could benefit from some examples of input before permanent defaults are in place. I had the script fail several times because I did not have proper inputs:
        input_collections_bias: ‘LATISS/calib’
        calib_collection: ‘LATISS/calib/u/ashugart/20210818.test’
        repo: ‘/repo/LATISS’
        certify_calib_begin_date: ‘2021-80-18’
        certify_calib_end_date: ‘2021-08-19’
        n_bias: 20 (or more)
- For troubleshooting, I would simply recommend to check the config first, but LOVE has so many outputs that we might not have them all characterized yet. I can add additional notes for the page if I find them. 
- Can it be made clear where it is possible to view the master frame?
- The LATISS scripting page should be populated in the same way the Comcam page is.

**Pull request merged**